composer.json

```diff
@@ -14,7 +14,7 @@
         "symfony/framework-bundle" : "~3.0"
     },
     "require-dev" : {
-        "atoum/atoum" : "master-dev",
+        "atoum/atoum" : "dev-master",
```

Comment:
As atoum is now [fully compatible with PHP 7.x](http://atoum.org/release/2017/02/22/release-3-0-0.html), I think we should explicitly use `~3.0`.